Makefile

```diff
@@ -38,22 +93,41 @@ dist/calicoctl: $(CALICOCTL_FILE) birdcl gobgp
 	 calico/build:latest \
 	 pyinstaller calicoctl.spec -ayF
 
-simple-binary:
+birdcl:
```

Comment:
`gobgp` and `birdcl` are dropped into project root on build. Can we volume mount pwd/dist instead?